### ISSUE TITLE
chat message disconnect

### ISSUE THREAD

**Issue by Wither-Strike** [Roeliefantje/minecraft_proxy#1]
for reference, this is in Roeliefantje/minecraft_proxy/minecraft_proxy.py
First, I want to address a typo in your code. on line 45 you accidentally put factory.connect_porty instead of factory.connect_port. not a huge deal, one backspace fixed it.
On to the main issue. I have no clue what causes this, but when I connect to a server through this proxy if I send a chat message I get the following error in minecraft: "Internal Exception: io.netty.handler.codec.DecoderException: Value (seemingly random number) is larger than limit 5"
Through a bit of doing the same thing over and over again, I got the number down to 50 once, I don't remember how though, but usually it's 7 digits long. however, this only appears to disconnect you if it's a chat as I can send commands like /me just fine. so I suppose technically it isn't a big deal as you can use /me or /say as replacements, but if you have the time I would say fixing this would be a good idea

**Comment by Wither-Strike** [comment]
slight update. I think the reason chat messages disconnect you but not commands is because they are different packets. when I type in a command like /me, /say, /awdaaszretgsz, or /verbose in the console output that is labeled as chat_command. with this in mind, I changed the function name packet_upstream_chat_message to packet_upstream_chat_command, the string "chat_message" to "chat_command", and the if statement from "/verbose" to "verbose" (because after a slight amount of testing, I found out that how it reads commands) now sending chat messages works. however, now I cannot send commands and get the following errors: in minecraft, I see "Protocol error" and in the console I see

Packet is too long: chat_command
Traceback (most recent call last):
  File "C:\Users\Wither_Strike\AppData\Local\Programs\Python\Python311\Lib\site-packages\quarry\net\protocol.py", line 249, in data_received
    raise ProtocolError("Packet is too long: %s" % name)
quarry.net.protocol.ProtocolError: Packet is too long: chat_command
